### ISSUE TITLE
configuration Instruction 

### ISSUE THREAD

**Issue by MichaelDuz** [zonefuenf/homebridge-enocean-zonefuenf#1]
I put an Enocean TCM 310 on my PI 4. LED lights up green. the plug in installed. what do I have to do now so that my switches are recognized?

**Comment by MichaelDuz** [comment]
Ich gehe davon aus,  dass ich den seriellen Port konfigurieren muss aber wie geht das?

**Comment by krukje** [comment]
Yes you need to configure the plugin in HomeBridge. And make sure the serial port is configured correctly and the device is recognized by your Pi 4.

This is the relevant part of my HomeBridge config.json:

```
    {
            "name": "EnOcean-Pi4",
            "port": "/dev/ttyUSB0",
            "logPresses": true,
            "logUnconfigured": true,
            "accessories": [
                {
                    "name": "fef4c1de Pi4",
                    "id": "fef4c1de",
                    "eep": "f6-02-01"
                }
            ],
            "platform": "enocean-zonefuenf"
        },
```

**Comment by MichaelDuz** [comment]
vielen Dank für die Rückmeldung. kannst Du mir sagen wie ich den Port richtig konfiguriere ind woran ich erkenne, dass der PI das Gerät erkannt hat?

**Comment by MichaelDuz** [comment]
Thank you for the feedback. can you tell me how to configure the port correctly and how do I know that the PI has recognized the device?

**Comment by SchelleLPH** [comment]
Guten Abend zusammen. Habe soweit alles am laufen. Homebridge auf dem Raspi, Plugin installiert, USB300 eingebunden. Alles läuft soweit. Nun habe ich den ersten Schalter eingebunden und wird auch im HomeKit angezeigt aber leider mit 4 unconfigured Buttons. Wie kann ich nun die Buttons zu schalten zubringen, wenn ich diese betätige? Kann nur Kurzbefehle hinterlegen. Wurden die Schalter hier nicht als solche erkannt? Danke im Voraus und VG
